### ISSUE TITLE
.rprofile interviews :: the next evolution

### ISSUE THREAD

**Issue by kellobri** [ropensci/unconf18#70]
I have a wild and crazy idea for iterating on the [.rprofile interview series](https://ropensci.org/tags/rprofile/) from last year. I'd like to continue doing .rprofiles this year, and anyone/everyone is welcome to join me in interviewing or being interviewer: [unconf17 Issue](https://github.com/ropensci/unconf17/issues/64)

But I'd also like to try something new. Possibly to be called `.Renviron Interviews`?

.Renviron Interviews focus less on personal career/life topics and more on personal workflows/demos. **_If you have a process or package or R-related life skill that you're highly opinionated about, I think it would make an awesome .Renviron piece._** I'm imagining the process would go something like: 

1. An interviewer and interviewee sit together in front of a computer.
2. They turn screen capture on.
3. They turn audio recording on.
4. The interviewee shows the interviewer how to do something or use something, using their favorite tools, workflow and opinionated development practices.
5. The interviewer asks questions about what's going on.
6. The interviewer transcribes and edits the audio into written form, and selectively chooses pieces of the recorded screen capture to edit into short GIFs.
7. The interviewee makes their own edits or suggestions.
8. You have an awesome .Renviron!

This could be a disaster! I've never tried it before! Does anyone want to participate? I'd love to hear feedback or nominations for interviews you'd like to see.

Props as always to @stefaniebutland for ideation and inspiration 🥇

**Comment by amoeba** [comment]
(I'd love to see that @karawoo!)

I like this idea. I was instantly reminded of Hadley's [video](https://www.youtube.com/watch?v=boKFxBniUH0) or sites like [emacsrocks](http://emacsrocks.com/) or [usethis](https://usesthis.com/). 👍 on transcribing to text and GIFs as a medium. Your idea is kind of a blend of these types of things which is great!

**Comment by kellobri** [comment]
Yes @amoeba - I was inspired by that video from Hadley and also [this one from Julia](https://youtu.be/evTuL-RcRpc). Resources like these are really cool, but they take a lot of planning and prep to put together - unless you're a super confident boss and can just rep 'em out. 

I'm hoping that the more laid back, _'I know someone will edit the heck out of this'_ approach will lower the scariness-to-entry for some folks and inspire the creation of more content like this.

**Comment by batpigandme** [comment]
If you need any assistance in the gif department, you know I'm down!

**Comment by kellobri** [comment]
**Summary: .rprofile and .Renviron interviews for rOpenSci**
.rprofile interviews are human interest pieces about members of our lovely R community
.Renviron interviews are a platform for showcasing opinionated workflows and use cases related to R

I'm interested in facilitating both these kinds of interviews at the unconf. Get @ me if you're interested in participating! - Kelly

@noamross

**Comment by maurolepore** [comment]
I nominate you @kellobri :) 
Maybe you can tell us how you manage to do so many things -- and so well --at the same time (relates to https://github.com/ropensci/unconf18/issues/60).

**Comment by jhollist** [comment]
@kellobri Love this idea!  I am an .rprofiles fanboy and would be equally enthusiastic about .renvirons!  While I am not at the unconf, if you are looking to add content, I'd be happy to provide an opinionated view of using R and Rstudio on a chromebook!   The mechanics of this with me sitting in RI might be a bit of challenge though...

**Comment by batpigandme** [comment]
All the more reason to ensure Kelly is at noreast'R!
-- 
Mara


--------------------------------------------------------
Mara Averick | maraaverick@gmail.com
http://www.linkedin.com/in/maraaverick

**Comment by jhollist** [comment]
Wheels are now turning!!!

**Comment by kellobri** [comment]
@jhollist I'd love this. I'm a Chromebook RStudio dabbler. Not yet opinionated. Let's make it happen!

**Comment by jhollist** [comment]
Great!  Maybe try something remote during the unconf (I've got time tomorrow around 3:30 PM EDT) or we could set something up later with more time to work on the details...  Might lean towards the later so you can focus on the on-site superstars!
